lib/modules/hide_downvote.js

```diff
@@ -0,0 +1,13 @@
+HNSpecial.settings.registerModule("hide_downvote", function () {
+  function hideDownvoteButton () {
+    var arrows = _.toArray(document.getElementsByClassName('votearrow'));
+
+    arrows.forEach(function(arrow) {
+      if (arrow.classList.contains('rotate180')) {
+        arrow.parentElement.remove();
+      }
+    });
+  }
+
+  hideDownvoteButton();
```

Comment:
You probably don't need to define this as a function in order to just call it once — have you considered just running the code directly within `registerModule`?